widgetssdk/src/main/res/values/strings.xml

```diff
@@ -23,6 +23,7 @@
     <string name="glia_visitor_code_error_title">Could not load the visitor code. Please try refreshing.</string>
     <string name="message_center_progress_bar_content_description">Sending</string>
     <string name="glia_chat_delivered">Delivered</string>
+    <string name="glia_chat_failed_to_deliver_retry">Failed to deliver. Tap to retry.</string>
```

Comment:
strings.xml should only contain the backwards compatible legacy strings. Newly added string values should be contained in new_strings.xml and not reference the default value in strings.xml

Reply:
Thank you for your attention. I missed this.
Fixed: d19ebb7bdc7e0716e4b06c96e4311abb0b474a90